src/linear_algebra/basis.lean

```diff
@@ -823,6 +823,10 @@ by rw [is_basis.constr, linear_map.range_comp, linear_map.range_comp, is_basis.r
 def module_equiv_finsupp (hv : is_basis R v) : M ≃ₗ[R] ι →₀ R :=
 (hv.1.total_equiv.trans (linear_equiv.of_top _ hv.2)).symm
 
+@[simp] theorem module_equiv_finsupp_apply (hv : is_basis R v) (i : ι) :
```

Comment:
This name suggests (to me) apply the equiv to a generic element, not just to `v i`.

Reply:
But `v i` is also an application.

Reply:
So... `_apply_apply`? More seriously... how about `module_equiv_finsupp_apply_basis`?

Reply:
Done